### ISSUE TITLE
Types change: Payload(s) [POST/GET]: `signers` (null | string[])

### ISSUE THREAD

**Issue by WietseWind** [XRPL-Labs/XUMM.NET.SDK#10]
## TL;DR:
Payloads have a new `meta` property: `signers`. The field can be `null` or an array containing between one and an arbitrary amount (currently hard capped by our backend at four, client doesn't have to enforce) XRPL account addresses.

When one or more account addresses are entered, XUMM forces signing with one of those accounts.

## Impact:
- [ ] Payload GET __types__ (`meta.signers`) (null or string[])
- [ ] Paylost POST __types__ (`options.signers`) (omit, null or string[])

## Commit to XUMM-SDK:
https://github.com/XRPL-Labs/XUMM-SDK/commit/8b5c484b23f774de70bcd3c6d95e87dd99d968b4

## Based on discussion:
https://github.com/XRPL-Labs/XUMM-Issue-Tracker/issues/384#issuecomment-1075098922

**Comment by DominiqueBlomsma** [comment]
@WietseWind 
Docs of `options.signers` should be added to the BODY PARAMS at https://xumm.readme.io/reference/post-payload.

`Signers` are added in [Add Payload Signers #13](https://github.com/XRPL-Labs/XUMM.NET.SDK/pull/13).

**Comment by DominiqueBlomsma** [comment]
`Signers` are included in release [v1.4.0](https://github.com/XRPL-Labs/XUMM.NET.SDK/releases/tag/v1.4.0).

**Comment by WietseWind** [comment]
> @WietseWind Docs of `options.signers` should be added to the BODY PARAMS at https://xumm.readme.io/reference/post-payload.
> 
> `Signers` are added in [Add Payload Signers #13](https://github.com/XRPL-Labs/XUMM.NET.SDK/pull/13).

Thanks for the heads up :) Added 🎉
